packages/expandable/tests/Expandable.test.tsx

```diff
@@ -1,66 +1,69 @@
 import React from "react";
-import { mount } from "enzyme";
+import { render } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
 import { createSerializer } from "@emotion/jest";
-import toJson from "enzyme-to-json";
 
 import { Expandable } from "../";
 
 expect.addSnapshotSerializer(createSerializer());
 
 describe("Sidebar", () => {
   it("renders", () => {
-    const component = mount(
+    const { asFragment } = render(
       <Expandable label="Label" isOpen={true}>
         Content
       </Expandable>
     );
-    expect(toJson(component)).toMatchSnapshot();
+    expect(asFragment()).toMatchSnapshot();
   });
-  it("toggles open and closed", () => {
-    const component = mount(
-      <Expandable label="Label" isOpen={true}>
-        Content
-      </Expandable>
+  it("toggles open and closed", async () => {
+    const { getByText, queryByText, getByTestId } = render(
+      <Expandable label="Label">Content</Expandable>
     );
 
-    expect(component.find("button").props()["aria-expanded"]).toBe(true);
-    component.find("button").simulate("click");
-    expect(component.find("button").props()["aria-expanded"]).toBe(false);
+    expect(queryByText("Content")).not.toBeVisible();
+    const labelEl = getByText("Label");
+    labelEl.click();
+    getByText("Content");
+    labelEl.click();
+    expect(queryByText("Content")).not.toBeVisible();
+
+    const chevronIcon = getByTestId("icon");
+    await userEvent.click(chevronIcon);
+    getByText("Content");
+    await userEvent.click(chevronIcon);
+    expect(queryByText("Content")).not.toBeVisible();
```

Comment:
can you write this as a separate test if it's a different element that controls the open/closed? Maybe we have two tests: "toggles open and close when label clicked" and "toggles open and close when chevron clicked"